README.md

```diff
@@ -1 +1,60 @@
 # dm-borrower-frontend
+
+The Borrower Frontend provides a website for Borrower to view and sign their
+mortgage deeds online.
+
+## Contents
+- [Usage](#usage)
+- [Getting Started](#getting-started)
+- [Acceptance Tests](#acceptance-tests)
+
+## Usage
+```
+GET     /health                         -- renders standard Gov UK template
+GET     /searchdeed                     -- borrower views deed landing page
+POST    /searchdeed/search              -- form POST the response is a html page which will display the deed or an error page
+```
+
+Install the requirements
+```
+pip install -r requirements.txt
+pip install -r requirements_test.txt
```

Comment:
You don't necessarily need requirements_test unless you're running the unit tests, either add a point about how to run the unit tests and include this there or take it out I would say